pkg/scanner/utils/utils.go

```diff
@@ -14,15 +15,34 @@ var (
 )
 
 func MatchVersions(currentVersion *version.Version, rangeVersions []string) bool {
```

Comment:
I know this change affects all `advisory.go` for each language, but `go-version` is no longer necessary. So, I feel it is the right way to replace `go-version` with `semver` in all places.

```suggestion
func MatchVersions(currentVersion *semver.Version, rangeVersions []string) bool {
```

Reply:
Done